.github/workflows/test_publish.yml

```diff
@@ -31,6 +30,7 @@ jobs:
         nox -e build
 
     - name: Publish distribution 📦 to Test PyPI
+      if: github.event.release.prereleased
```

Comment:
```suggestion
```

If `types: [prereleased]` works (from other suggestion), we can remove this if conditional.

Reply:
done